README.md

```diff
@@ -518,6 +519,37 @@ source dbt_packages/fivetran_utils/generate_docs.sh '../dbt_apple_search_ads_sou
 **Args:**
 * `package` (required): Name of the package; include whether package is source or not
 
+----
+### get_column_names_only ([source](macros/get_column_names_only.sql))
+This macro is used in the `generate_models.sh` script to further the `staging_models_automation` macro. This macro outputs all columns from the specified table, allowing `generate_models.sh` to prefill column fields in the final select statement.
+
+Note this will retain the timestamp from the built-in formatting update from dbt 1.0.0. Therefore in the staging model resulting from `generate_models.sh`, you will need to manually delete the timestamp.
+
+**Usage:**
+```bash
+dbt run-operation get_column_names_only --args '{table_name: [table_name], schema_name: [schema_name], database_name: [database_name]}' | tail -n +2 
```

Comment:
When reading this I would imagine I need to add brackets when inserting my arguments. So I would think I need to do:
```bash
dbt run-operation get_column_names_only --args '{table_name: [log], schema_name: [fivetran_log], database_name: [my-database]}'
```
However, I think we need to use single quotes instead. Can you update this to be similar to a real world example of how to use this.

Reply:
Also, is the `tail -n +2` needed in the usage?

Reply:
Updated to use real example!

Also, so this is what it looks like without that formatting at the end. But you're right in that we don't actually need to include the formatting bit in the macro description itself, so I'll remove it here.
```
21:19:03  Running with dbt=1.0.0
21:19:03  Unable to do partial parsing because a project config has changed
21:19:05          _fivetran_synced,
        connector_id,
        event,
        id,
        message_data,
        message_event,
        process_id,
        sequence_number,
        sync_id,
        time_stamp,
        transformation_id

```